demo/demo.html

```diff
@@ -1,12 +1,10 @@
 <head>
     <title>GeoTIFFTileSource Demo</title>
-    <!-- <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/openseadragon/3.1.0/openseadragon.min.js"></script> -->
-    <!-- <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/geotiff"></script> -->
-    <script type="text/javascript" src="../lib/openseadragon.js"></script>
-    <script type="text/javascript" src="../lib/geotiff.js"></script> 
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/openseadragon/3.0.0/openseadragon.min.js"></script>
```

Comment:
I see you've changed this to use OSD 3.0.0. I presume this was done to test the monkey patch. I think we should make it the current version of OSD (4.1.1) now that testing is complete.